presto-jdbc/src/main/java/com/facebook/presto/jdbc/ColumnInfo.java

```diff
@@ -227,6 +233,8 @@ private static int getType(TypeSignature type)
                 return Types.DATE;
             case "decimal":
                 return Types.DECIMAL;
+            case "float":
+                return Types.REAL;
```

Comment:
I was just looking through the SQL spec and it seems like REAL should also be the name of the type on the server side (see section 6.1.30). How did you chose the FLOAT name? It seems like we should rename it to REAL